app/assets/stylesheets/nav.scss

```diff
@@ -21,36 +48,75 @@ $button-color: #1E90FF;
 
   input {
     width:280px;
+    border-radius: 5%;
   }
+
   label {
       color: orange;
       font-weight: bold;
+
+      span{
```

Comment:
Line should be indented 4 spaces, but was indented 6 spaces<br>Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space